test/smoke/appServers/global-resources/httpheaders_applicationinsights.json

```diff
@@ -0,0 +1,25 @@
+{
+  "connectionString": "InstrumentationKey=00000000-0000-0000-0000-0FEEDDADBEEF;IngestionEndpoint=http://host.docker.internal:6060/",
+  "role": {
+    "name": "testrolename",
+    "instance": "testroleinstance"
+  },
+  "preview": {
+    "captureHttpServerHeaders": {
```

Comment:
not sure how server headers will be helpful for customers?  can you share a use case for this?

Reply:
is it customer's server or app insights backend server? it's not clear to me.